src/client/browser/index.js

```diff
@@ -0,0 +1,45 @@
+import COMMAND from '../../browser-connection/command';
+import hammerheadApi from './deps/hammerhead';
+
+
+const HEARTBEAT_INTERVAL = 30 * 1000;
+
+
+//Utils
+var XMLHttpRequest = hammerheadApi ? hammerheadApi.NativeMethods.XMLHttpRequest : window.XMLHttpRequest;
+
+function sendXHR (url, callback) {
+    var xhr = new XMLHttpRequest();
```

Comment:
done. We use the `es6-promise` module on the client-side and the `promise` module on the server-side. We should switch to the `es6-promise` everywhere (like in the `testcafe-hammerhead` repo), but not in this pr